### PR TITLE
feat(msg-parser/huobi): Implemented `parse_candlestick()` for huobi

### PR DESCRIPTION
**Pull request by iyangdianfeng**
Adding Huobi message resolution

### CODE REVIEW

**Review by soulmachine — CHANGES_REQUESTED** [review]
Please merge main branch by running `git checkout binance_parse_candlestick && git merge main`

**Review by soulmachine — CHANGES_REQUESTED** [review]
`cargo fmt` failed, please run `cargo fmt` locally

File: crypto-msg-parser/src/exchanges/huobi/huobi_spot.rs [hunk 1]
```diff
@@ -195,3 +195,74 @@ pub(super) fn parse_bbo(msg: &str) -> Result<Vec<BboMsg>, SimpleError> {
...
+        "1year" => (1, "Y"),
+        _ => {
+            return Err(SimpleError::new(format!(
+                "Unknown huobi period error {}",
+                period
+            )))
+        }
+    };
+
+    // obj.data.k
```
Comment (pan93412) [inline crypto-msg-parser/src/exchanges/huobi/huobi_spot.rs:247]: Better to remove this dead code.

**Review by pan93412 — CHANGES_REQUESTED** [review]

File: crypto-msg-parser/src/exchanges/huobi/huobi_spot.rs (hunk 1 above)
Comment (soulmachine) [reply crypto-msg-parser/src/exchanges/huobi/huobi_spot.rs]: Removed

**Review by soulmachine — APPROVED** [review]

**Pull request merged**